contrib/cirrus/lib.sh

```diff
@@ -232,6 +230,7 @@ install_testing_deps() {
     echo "Installing latest upstream version of BATS"
     ooe.sh git clone https://github.com/bats-core/bats-core.git /tmp/bats
     cd /tmp/bats
+	echo "Testing against bats commit: $(git rev-parse HEAD)"
```

Comment:
if you're keeping this long term, you might want to tweak the space/tab issue here.